llm/config/config.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+import (
+	"github.com/joho/godotenv"
+)
+
+type Config struct {
+	OllamaModels []string
+	OllamaURL    string
+
+	TimeoutSeconds int
+}
+
+var (
+	config     *Config
+	configOnce sync.Once
+	configErr  error
+)
+
+func Load(envFile string) (*Config, error) {
+	configOnce.Do(func() {
+		config = &Config{}
+		err := godotenv.Load(envFile)
+		if err != nil {
+			configErr = fmt.Errorf("error loading .env file: %v", err)
+			return
+		}
+
+		modelsEnv := os.Getenv("OLLAMA_MODELS")
+		if modelsEnv == "" {
+			configErr = fmt.Errorf("error: OLLAMA_MODELS environment variable is not set")
+			return
+		}
+
+		modelsList := strings.Split(modelsEnv, ",")
+		for i, model := range modelsList {
+			modelsList[i] = strings.TrimSpace(model)
```

Comment:
```suggestion
			modelsList[i] = strings.TrimSpace(model)
```
config 里面已经 TrimSpace 过了 调用cfg.OllamaModels 的时候为什么还要再 TrimSpace 一遍